spec/views/work_package/show.html.erb_spec.rb

```diff
@@ -28,22 +28,28 @@
 
 require 'spec_helper'
 
-describe 'work_packages/show', :type => :view do
+describe 'work_packages/show', type: :view do
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.